lib/engines/content_block_manager/app/views/content_block_manager/content_block/editions/workflow/review_update.html.erb

```diff
@@ -0,0 +1,71 @@
+<% content_for :context, "Create a content block" %>
+<% content_for :title, "Review #{@content_block_edition.block_type.humanize.downcase}" %>
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", {
+    href: content_block_manager.content_block_manager_content_block_documents_path,
+  } %>
+<% end %>
+
+<% if @error_summary_errors %>
+  <%= render "govuk_publishing_components/components/error_summary", {
+    title: "There is a problem",
+    items: @error_summary_errors,
+  } %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <%= render(
+          ContentBlockManager::ContentBlockEdition::Show::ConfirmSummaryListComponent.new(
```

Comment:
the final design calls for extra columns in the update flow, but as this PR has dragged on so much I figured these could be added in next PR.